src/LibPQ.jl

```diff
@@ -26,37 +33,36 @@ const LOGGER = getlogger(@__MODULE__)
 
 function __init__()
     INTERVAL_REGEX[] = _interval_regex()
-    Memento.register(LOGGER)
+    return Memento.register(LOGGER)
 end
 
 # Docstring template for types using DocStringExtensions
-@template TYPES =
-    """
-        $(TYPEDEF)
+@template TYPES = """
+                      $(TYPEDEF)
 
-    $(DOCSTRING)
+                  $(DOCSTRING)
 
-    ## Fields:
+                  ## Fields:
 
-    $(TYPEDFIELDS)
-    """
+                  $(TYPEDFIELDS)
+                  """
```

Comment:
This seems like the only particularly odd change/unfortunate change

Reply:
I think this is something we should add to the style guide (to prefer what previously existed). 

The placement of the rest of the text and the closing triple quote relative to the opening triple quote makes no difference to the content of the string.

Reply:
yes please!
https://github.com/invenia/BlueStyle/issues/60

Reply:
https://github.com/invenia/BlueStyle/pull/71

Reply:
Interestingly, the formatter seems to have been confused here by where the opening triple quotes

By BlueStyle, this should be 
```julia
@template TYPES = """
        (TYPEDEF)

    (DOCSTRING)

    ## Fields:
    (TYPEDFIELDS)
    """
```
And JuliaFormatter (v0.9) would leave that as is
```julia
julia> str = """
       @template TYPES = \"\"\"
               (TYPEDEF)

           (DOCSTRING)

           ## Fields:
           (TYPEDFIELDS)
           \"\"\"
           """;

julia> println(format_text(str, BlueStyle()))
@template TYPES = """
        (TYPEDEF)

    (DOCSTRING)

    ## Fields:
    (TYPEDFIELDS)
    """
```

But the actual code here right now has the opening triple-quot on the line below i.e. it is
```julia
@template TYPES =
    """
        (TYPEDEF)

    (DOCSTRING)

    ## Fields:
    (TYPEDFIELDS)
    """
```
which seems to confuse the formatter (?), which changes this too
```julia
julia> str = """
       @template TYPES =
           \"\"\"
               (TYPEDEF)

           (DOCSTRING)

           ## Fields:
           (TYPEDFIELDS)
           \"\"\"
           """;

julia> println(format_text(str, BlueStyle()))
@template TYPES = """
                      (TYPEDEF)

                  (DOCSTRING)

                  ## Fields:
                  (TYPEDFIELDS)
                  """
```

Perhaps @domluna knows if this is intended behaviour (which just needs overloading for BlueStyle) or if something funky is going on?

Reply:
this is intended, the idea being that the indentation of the string changes based on where the tripe quote opener is moved. This allows it to be moved around intact.

In this case (TYPEDEF) is 4 spaces in behind the opening quote so when the quote is moved up a line, (TYPEDEF) follows to be 4 spaces behind the new position.

Reply:
okay, cool - thanks - makes sense! BlueStyle prefers something different, so in time it'd be nice to support that for `BlueStyle()` https://github.com/domluna/JuliaFormatter.jl/issues/283#issuecomment-687795763